src/components/projects/hooks.js

```diff
@@ -20,16 +20,16 @@ async function getProjectAge(id, events, setState){
   }
 }
 
-async function getProjectAssets(projectId, setState){
+async function getProjectAssets(projectId, setState, debug=false){
   EmbarkJS.onReady(async (_err) => {
     const projectInfo = await LiquidPledging.methods.getPledgeAdmin(projectId).call()
     const CID = projectInfo.url.split('/').slice(-1)[0]
-    console.log({CID, projectInfo, ipfs})
+    if (debug) console.log({CID, projectInfo, ipfs})
```

Comment:
We could use an env var for that like `process.env.DEBUG` or something like that, that way we don't have to pass a parameter

Reply:
we could, we have to see why `process` is not being passed through currently.

Reply:
Oh, my bad, it's because this is not node code, it's browser code. We'd need to define it in window or document instead